cmake/xacc-config.cmake.in

```diff
@@ -20,11 +20,7 @@ if(NOT TARGET format)
 endif()
 
 # Set version info
-set(XACC_VERSION_MAJOR @MAJOR_VERSION@)
-set(XACC_VERSION_MINOR @MINOR_VERSION@)
-set(XACC_VERSION_PATCH @PATCH_VERSION@)
 set(XACC_VERSION_SUFFIX @XACC_BUILD_VERSION@)
-set(XACC_VERSION "@MAJOR_VERSION@.@MINOR_VERSION@.@PATCH_VERSION@-@XACC_BUILD_VERSION@")
```

Comment:
These are set by `find_package()` automatically. See the end of this section: https://cmake.org/cmake/help/latest/command/find_package.html#config-mode-version-selection